src/proto/tactic.proto

```diff
@@ -28,7 +27,7 @@ message Tactic
         PassDefenderTactic pass_defender                    = 15;
         ShootSkillTactic shoot_skill                        = 16;
     }
-    reserved 4, 5, 11;
+    reserved 2, 4, 5, 7, 11;
```

Comment:
these aren't logged in replays, so we don't __need__ to reserve them but we can for protobuf specs anyways